<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>CurveDialogWidget</class>
  <widget class="QWidget" name="CurveDialogWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>775</width>
        <height>369</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <item row="1" column="0" >
        <widget class="CurveAppearanceWidget" name="_curveAppearance" />
      </item>
      <item row="1" column="1" >
        <widget class="CurvePlacementWidget" name="_curvePlacement" />
      </item>
      <item rowspan="1" row="2" column="0" colspan="2" >
        <widget class="QCheckBox" name="_checkBoxIgnoreAutoscale" >
          <property name="text" >
            <string>Ignore in automatic axes range calculations</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="0" column="0" colspan="2" >
        <widget class="Q3GroupBox" name="GroupBox9_4" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Curve Contents</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item row="0" column="0" >
              <widget class="QLabel" name="TextLabel1_6_3" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>X axis vector:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="5" column="0" colspan="2" >
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <widget class="QCheckBox" name="_checkBoxXMinusSameAsPlus" >
                    <property name="text" >
                      <string>Use +X error bar for -X</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QCheckBox" name="_checkBoxYMinusSameAsPlus" >
                    <property name="text" >
                      <string>Use +Y error bar for -Y</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="spacer33" >
                    <property name="sizeHint" >
                      <size>
                        <width>71</width>
                        <height>31</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
            <item row="0" column="2" >
              <widget class="QLabel" name="TextLabel1_6_2_5" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>Y axis vector:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="1" column="0" colspan="4" >
              <widget class="Line" name="line1_3" >
                <property name="frameShape" >
                  <enum>QFrame::HLine</enum>
                </property>
                <property name="frameShadow" >
                  <enum>QFrame::Sunken</enum>
                </property>
              </widget>
            </item>
            <item rowspan="2" row="2" column="0" colspan="1" >
              <widget class="QLabel" name="TextLabel1_6_2_2_2" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>+X error bar:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring></cstring>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="0" >
              <widget class="QLabel" name="textLabelXMinus" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>-X error bar:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring></cstring>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="2" >
              <widget class="QLabel" name="textLabelYMinus" >
                <property name="text" >
                  <string>-Y error bar:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="3" >
              <widget class="VectorSelector" name="_yVector" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item row="3" column="3" >
              <widget class="VectorSelector" name="_yError" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item row="4" column="3" >
              <widget class="VectorSelector" name="_yMinusError" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item row="3" column="2" >
              <widget class="QLabel" name="textLabel1" >
                <property name="text" >
                  <string>+Y error bar:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="1" >
              <widget class="VectorSelector" name="_xMinusError" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item rowspan="2" row="2" column="1" colspan="1" >
              <widget class="VectorSelector" name="_xError" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="VectorSelector" name="_xVector" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>_xVector</tabstop>
    <tabstop>_yVector</tabstop>
    <tabstop>_xError</tabstop>
    <tabstop>_yError</tabstop>
    <tabstop>_xMinusError</tabstop>
    <tabstop>_yMinusError</tabstop>
    <tabstop>_checkBoxXMinusSameAsPlus</tabstop>
    <tabstop>_checkBoxYMinusSameAsPlus</tabstop>
    <tabstop>_curveAppearance</tabstop>
    <tabstop>_curvePlacement</tabstop>
    <tabstop>_checkBoxIgnoreAutoscale</tabstop>
    <tabstop>_curveAppearance</tabstop>
    <tabstop>_curvePlacement</tabstop>
  </tabstops>
  <includes>
    <include location="global" >kst_export.h</include>
  </includes>
</ui>
